### DISCUSSION TITLE
blog/ps3

### DISCUSSION THREAD

**Discussion by giscus** [NotNite/giscus#7] [Announcements]
# blog/ps3

https://notnite.com/blog/ps3

**Comment by rewrite** [comment]
:cat2:

**Comment by lokinmodar** [comment]
🫦

**Comment by BEZEL-fA** [comment]
Thank you for writing this.

I am interested in analyzing a certain game, and I need to understand the contents of the EBOOT to see how it is actually processed.

I am a complete beginner, but thanks to people like you, I can take on more challenges.

**Comment by Erarnitox** [comment]
This was a great read! The PS3 is where it all started for me and reads like this make me feel old and nostalgic.

A very cool and informative read! Glad you enjoyed your time on the PS3!

**Comment by Martvse** [comment]
I started developing for the PlayStation 3 toward the end of last year. The learning curve is steep, but highly rewarding. I built my first SPRX by studying YouTube tutorials, examining older Call of Duty mod-menu SPRX source code and projects, and drawing some inspiration from ChatGPT.

At this point, my SPRX runs reliably, and I am able to read from and modify memory. However, development currently still requires DEX firmware. I hook the SPRX into the EBOOT process using webMAN. I am unsure whether this approach will work on retail firmware with a jailbreak, as memory addresses may differ or shift. At the moment, I do not fully understand if this is the case, nor the correct way to account for or resolve it.

It is unfortunate that so much knowledge in this area is gradually being lost—something I can strongly relate to.
